bundles/org.openhab.binding.fronius/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -272,4 +315,31 @@
 		<state pattern="%.2f %unit%" readOnly="true"></state>
 	</channel-type>
 
+	<channel-type id="ohmpilot_powerreal">
+		<item-type>Number:Power</item-type>
+		<label>Power</label>
+		<description></description>
+		<state pattern="%.2f %unit%" readOnly="true"></state>
+	</channel-type>
+	<channel-type id="ohmpilot_energy">
+		<item-type>Number:Energy</item-type>
+		<label>Energy</label>
+		<description></description>
+		<state pattern="%.2f %unit%" readOnly="true"></state>
+	</channel-type>
+	<channel-type id="ohmpilot_temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description></description>
+		<state pattern="%.2f %unit%" readOnly="true"></state>
+	</channel-type>
+	<channel-type id="ohmpilot_state">
+		<item-type>Number</item-type>
+		<label>State</label>
+		<description></description>
```

Comment:
Please set the description or remove the tag in case you think there is no need to describe the channel.

Please do the same for already existing channels.